python/src/aiconfig/editor/client/src/shared/prompt_schemas/DalleImageGenerationParserPromptSchema.ts

```diff
@@ -0,0 +1,35 @@
+import { PromptSchema } from "../../utils/promptUtils";
+
+export const DalleImageGenerationParserPromptSchema: PromptSchema = {
+  // "n", "quality", "response_format", "size", "style"
```

Comment:
delete?

Reply:
Let's also add comment in code saying this is to match the types from `/opt/homebrew/Caskroom/miniconda/base/envs/aiconfig/lib/python3.12/site-packages/openai/resources/images.py:`

Separate note: this also kind of fits with general theme (not P1 or even P2) of us should maybe define schema on backend in 1 centralized place?